custom_model_runner/datarobot_drum/drum/description.py

```diff
@@ -4,6 +4,6 @@
 This is proprietary source code of DataRobot, Inc. and its affiliates.
 Released under the terms of DataRobot Tool and Utility Agreement.
 """
-version = "1.10.11"
+version = "1.10.12"
```

Comment:
Thanks so much for making these updates.

I was wondering about merging this PR I made before the next release?
https://github.com/datarobot/datarobot-user-models/pull/858